index.html

```diff
@@ -3,12 +3,48 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css"
+    />
+
     <link rel="stylesheet" href="/styles/style.css" />
     <title>ThemeMascot Covid-19 UI</title>
   </head>
   <body>
     <nav id="nav" class="nav">
-      <div class="container-nav"></div>
+      <div class="container-nav">
+        <div class="brand">
+          <a href="index.html">
+            <img
+              src="./assets/logo-white.png"
+              alt="ThemeMascot"
+              class="brand-img"
+            />
+          </a>
+        </div>
+        <input type="checkbox" class="toggler" id="toggler" />
+        <div class="menu-wrap">
```

Comment:
Noticing a blue background when the menu is in dropdown. That's not consistent with what the actual site you are replicating. Let's fix that.

Also nice touch with switching the hamburger menu into a cancel icon when it's dropped down. (Makes clicking a bit glitchy though as it seems to take some time to respond).

Reply:
Will work on it